calcom/cal.com
# Add links to Zapier templates in installed apps

## What does this PR do?

- Adds new "Automation" tab to Installed apps
- Adds a new component for App specific settings and used it for Zapier to show the templates and link them to the actual Zapier templates. 
- Small mobile design fixes for installed apps 

Fixes #4048

Loom: https://www.loom.com/share/f84a6e5e1f7c4d7fbf61c03612bbfe6b

**Environment**: Staging(main branch)

## Type of change


## How should this be tested?

- Install Zapier 
- Go to installed apps -> automation
- See the new Zapier templates
- Click on 'Use zap' and check if you are redirected to the correct template
